sprint-mission-2/src/main/java/com/sprint/mission/discodeit/service/ChannelService.java

```diff
@@ -0,0 +1,21 @@
+package com.sprint.mission.discodeit.service;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ChannelService {
+    public Channel createChannel(String name, User root);
```

Comment:
인터페이스 메서드 내 접근제어자는 필요없어서 제거하는게 좋을 것 같아요.  

```suggestion
    Channel createChannel(String name, User root);
```

인터페이스의 접근제어자는 왜 필요없을까요?

Reply:
인터페이스를 상속받은 구현체마다 상속받은 메서드에 접근제어가 다를 수 있기 때문입니다?

Reply:
정확히는 인터페이스는 기본적으로 접근제어자가 public이어서예요,  

인터페이스 자체가 메서드 시그니처를 정의하는 역할을 담당하기에 인터페이스는 기본적으로 접근제어자가 public입니다ㅎㅎ